### DISCUSSION TITLE
4AD1CA6A-93D2-5F94-A72E-7E5B6042E8B1

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#156] [General]
# 4AD1CA6A-93D2-5F94-A72E-7E5B6042E8B1

Tau Bootis Ab is an exoplanet discovered in 1996 and located 51 light-years away from Earth. It is a hot Jupiter that is four times more massive than Jupiter and that orbits very close to its parent star.

https://celestia.mobi/addon?item=4AD1CA6A-93D2-5F94-A72E-7E5B6042E8B1

**Comment by exocritic5457** [comment]
What's all the talk about Tau Boötis A being tidally locked to Ab about? is it the star itself that's locked or something else like its magnetic field
